examples/demo_all_cisco.pp

```diff
@@ -38,6 +38,8 @@
   include ciscopuppet::cisco::demo_pim
   include ciscopuppet::cisco::demo_portchannel
   include ciscopuppet::cisco::demo_snmp
+  #include ciscopuppet::cisco::demo_stp_bd
```

Comment:
I think this is outdated as we discussed. I will try the way you asked and update the test code